apps/settings/js/panels/root/privacy_items.js

```diff
@@ -0,0 +1,24 @@
+define(function(require) {
```

Comment:
Why do you need to create one more almost identical function? Why not to use parameters to define e.g. list of updatable elements?